src/hc-scripts/download-cvs.ts

```diff
@@ -1,13 +1,15 @@
 import * as fs from 'fs';
 import * as https from 'https';
+import * as _ from 'lodash';
```

Comment:
If lodash exports `shuffle` it would be better to import that specifically and then be able to use `_` for an empty parameter

Reply:
+1 on importing shuffle directly.  I think we should be using () for anonymous functions with no parameters, or parameters prefixed with underscores, since TypeScript doesn't treat `_` specially (like OCaml for example).

Examples:

`() => 1`
`(_x: A, y: B) => y`
`(x: A, _y: B) => x` or `(x:A) => x`

Reply:
Okay in that case be should add a linter rule since this has been much used (by me) on the advice of @henry-thompson.

Reply:
None available it seems

Reply:
To be clear, this is a real usage, a bit like one-line brace-less `if` statements, just not one used by much of the JS community it seems.